tools/sidekick/app.js

```diff
@@ -323,11 +323,19 @@
    * @param {Sidekick} sk The sidekick
    */
   function checkForUpdates(sk) {
+    // check for wrong byocdn config
+    // https://github.com/adobe/helix-pages/issues/885
+    if (sk.config.byocdn && !/^www.*\.adobe\.com$/.test(sk.config.host)) {
```

Comment:
Do we really need to hardcode www.adobe.com here ?

Reply:
If we want to auto-heal user configs, yes. All `*adobe.com` prod hosts except `www*.adobe.com` are affected. I;m open to better solutions!